### PR TITLE
Add build script for filebeat version 5.6.16

### PR DESCRIPTION
**Pull request by snehlatamohite**
Build Script for file beat version 5.6.16 and creates rpm

### CODE REVIEW

File: f/filebeat/filebeat_v5.6.16_ubi.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+#! /bin/bash
+dnf -y install \
+        http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/Packages/centos-linux-repos-8-3.el8.noarch.rpm \
```
Comment (seth-priya) [inline f/filebeat/filebeat_v5.6.16_ubi.sh:16]: which dependencies are coming from these? aren't they present in EPEL?

File: f/filebeat/filebeat_v5.6.16_ubi.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+%prep
+yum install -y wget git make gcc-c++ python3-virtualenv
+
+wget https://golang.org/dl/go1.10.2.linux-ppc64le.tar.gz
+tar -zxvf go1.10.2.linux-ppc64le.tar.gz
+
+export GOPATH=~/rpmbuild/BUILD/go
+export PATH=$PATH:$GOPATH/bin
+mkdir -p ${GOPATH}/src/github.com/elastic
+cd ${GOPATH}/src/github.com/elastic
```
Comment (seth-priya) [inline f/filebeat/filebeat_v5.6.16_ubi.sh:61]: shouldn't we just use "go get" then? we won't need to manually create the paths

File: f/filebeat/filebeat_v5.6.16_ubi.sh [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+git clone https://github.com/elastic/beats.git
+cd beats/filebeat
+git checkout ${VERSION}
+make
+
```
Comment (seth-priya) [inline f/filebeat/filebeat_v5.6.16_ubi.sh:66]: aren't there any tests?

Comment (snehlatamohite) [reply f/filebeat/filebeat_v5.6.16_ubi.sh:66]: Thanks for review comments. I will ask Saurabh to work on review comments and will revert.

**Comment by snehlatamohite** [comment]
@saurabh-gore  - Please work on review comments.

**Comment by Sapana-Khemkar** [comment]
Closing as latest version script is already available

**Pull request closed without merging**